src/main/java/frc/robot/subsystems/arm/ArmPositions.java

```diff
@@ -0,0 +1,174 @@
+        SmartDashboard.putNumber("Elbow Standing Cone", Positions.ELBOW_PICK_UP_CONE);
+        // Arm Intake Cube
+        SmartDashboard.putNumber("Arm Cube", Positions.ARM_PICK_UP_CUBE);
+        SmartDashboard.putNumber("Elbow cube", Positions.ELBOW_PICK_UP_CUBE);
+        // Score Cube Mid
+        SmartDashboard.putNumber("Arm Cube Mid", Positions.ARM_SCORE_CUBE_MID);
+        SmartDashboard.putNumber("Elbow Cube Mid", Positions.ELBOW_SCORE_CUBE_MID);
+        // Score Cube High
+        SmartDashboard.putNumber("Arm Cube High", Positions.ARM_SCORE_CUBE_HIGH);
+        SmartDashboard.putNumber("Elbow Cube High", Positions.ELBOW_SCORE_CUBE_HIGH);
+        // Score Cone Mid
+        SmartDashboard.putNumber("Arm Cone Mid", Positions.ARM_SCORE_CONE_MID);
+        SmartDashboard.putNumber("Elbow Cone Mid", Positions.ELBOW_SCORE_CONE_MID);
+        // Score Cone High
+        SmartDashboard.putNumber("Arm Cone High", Positions.ARM_SCORE_CONE_HIGH);
+        SmartDashboard.putNumber("Elbow Cone High", Positions.ELBOW_SCORE_CONE_HIGH);
+        // Elbow Idle Position
+        SmartDashboard.putNumber("Elbow Idle", Positions.ELBOW_IDLE);
+    }
+
+    public static class Positions {
+        // Arm Stowed Positions
+        public static double ARM_STOWED = .55;
+        public static double ELBOW_STOWED = .79;
+        public static double ARM_PICK_UP_TCONE = .35;
+        public static double ELBOW_PICK_UP_TCONE = .529;
+        public static double ARM_PICK_UP_CONE = 0.1658;
+        public static double ELBOW_PICK_UP_CONE = 0.31;
+        public static double ARM_PICK_UP_CUBE = 0.325;
+        public static double ELBOW_PICK_UP_CUBE = 0.60;
+        public static double ARM_SCORE_CUBE_LOW = 0.166;
+        public static double ELBOW_SCORE_CUBE_LOW = 0.4;
+        public static double ARM_SCORE_CONE_LOW = 0.172;
+        // .7345
+        public static double ARM_SCORE_CONE_MID = 0.74;
+        public static double ELBOW_SCORE_CONE_LOW = 0.356;
+        // .58
+        public static double ELBOW_SCORE_CONE_MID = 0.58;
+
+        public static double ARM_SCORE_CUBE_MID = 0.71;
+        public static double ELBOW_SCORE_CUBE_MID = 0.62;
+        // .0185
+        public static double ARM_SCORE_CONE_HIGH = 0.0185;
+        // .32
+        public static double ELBOW_SCORE_CONE_HIGH = 0.32;
+
+        public static double ARM_SCORE_CUBE_HIGH = 0.2;
+        public static double ELBOW_SCORE_CUBE_HIGH = 0.62;
+
+        public static double ARM_PICK_UP_SHELF = 0.445;
+        public static double ELBOW_PICK_UP_SHELF = 0.83;
+        // Arm back cone pick up scoring position
+        public static double ARM_BACK_PICKUP = 0.0;
+        public static double ELBOW_BACK_PICKUP = 0.0;
+        public static double ARM_SPECIAL_IDLE = 0.0;
+        public static double ELBOW_SPECIAL_IDLE = 0.0;
+        // Elbow Idle position
+        public static double ELBOW_IDLE = .83;
+        // public static double ARM_STOWED = .55;
```

Comment:
Should all these commented out constants be deleted?

Reply:
comments have been deleted